templates/component-pages/component-name/usage.mdx

```diff
@@ -0,0 +1,41 @@
+---
+# Leave the frontmatter as is
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  label: Usage
+data:
+  $ref: ./#/data
+---
+
+### Using the Component Name component
+
+Usage information for this component. Copy the content from the content
+template's "How to use" section here.
+
+### Import
+
+{/* Update the text and code snippet below as needed: */}
+
+To import Component Name from the core Salt package, use:
+
+```
+import { ComponentName } from "@salt-ds/core";
+```
+
+### Props
+
+{/* Update componentName below as needed */}
+
+<PropsTable componentName="ComponentName" />
```

Comment:
Should we also add a note about the optional `packageName` prop for non-core components?

Reply:
Yeah good idea. Just did it